src/utilities/validateUserInput.js

```diff
@@ -108,6 +117,25 @@ class Validate {
       });
     }
   }
+
+  /**
+   * Validate ratings Input
+   * @param {integer} req - request
+   * @param {object} res - Response object
+   *  @param {callback} next - The callback that passes the request
+   * to the next handler
+   * @returns {object} - server responce
+   */
+  static validateRating(req, res, next) {
+    const { rating } = req.body;
+    if ((rating > 5) || (rating < 1)) {
+      return res.status(400).json({
+        success: false,
```

Comment:
You are also not trimming the user-input, rating, before validating it. Use the "trimvalues()" method to trim the user-input (rating) before validating it.

Reply:
Thanks for the feedback. The code has been refactored.